generators/app/templates/src/projectName.scss

```diff
@@ -0,0 +1,25 @@
+@import 'node_modules/terra-mixins/src/terra-mixins';
```

Comment:
I am instead including it as a default them added by the sass loader in webpack.config.js